libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -680,7 +689,8 @@ fun readableMapOf(reverseSwapPairInfo: ReverseSwapPairInfo): ReadableMap {
             "feesHash" to reverseSwapPairInfo.feesHash,
             "feesPercentage" to reverseSwapPairInfo.feesPercentage,
             "feesLockup" to reverseSwapPairInfo.feesLockup,
-            "feesClaim" to reverseSwapPairInfo.feesClaim
+            "feesClaim" to reverseSwapPairInfo.feesClaim,
+            "feesTotalEstimated" to reverseSwapPairInfo.feesTotalEstimated
```

Comment:
Needs renaming to `totalEstimatedFees`

Reply:
Done in f02c462fb1406ad4f07a94a1a9dfb440e140bec1